packages/sdk/test/integration/gc_test.ts

```
@@ -419,32 +731,107 @@
+    );
 
     await client2.attach(doc2, { syncMode: SyncMode.Manual });
+    assert.equal(
+      versionVectorHelper(doc2.getVersionVector(), [
+        { actor: client1.getID()!, lamport: Long.fromNumber(3) },
+        { actor: client2.getID()!, lamport: Long.fromNumber(4) },
+      ]),
+      true,
+    );
     assert.equal(doc2.getGarbageLen(), 1);
     doc2.update((root) => (root.point.x = 2));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)